website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+
+**3. Responsive Design**
+
+On account of the effort to improve the User Experience, we have made the UI more responsive to window size changes. This includes animations as well as consistent design across all sections of the application.
+
+**4. Registries**
+
+We have also updated the UI of the page where we add/update registries in ‘Settings’.
+
+![registries](img/podman-desktop-release-0.10/registries.png)
+
+**5. Font and text**
+
+Other improvements include making font colors across the application consistent. Significant efforts have been made to display information correctly whether they are container logs or other text information.
+
+## Major enhancements and fixes
+
+**1. More options to configure containers**
+
+Until now, we could only specify port binding when building images to start containers. With the latest release, it includes several options such as configuring volumes, environment variables, some more advanced options including restart policy and options regarding networking and security. You can also reuse a network when creating containers.
+
+![create container basic](img/podman-desktop-release-0.10/create-container-basic.png)
+
+![create container security](img/podman-desktop-release-0.10/create-container-security.png)
+
+**2. Notify Proxy Changes**
+
+Updates to the proxy are now propagated to the listeners.
+
+**3. Automate the creation of the  Podman Socket when installing from Flatpack**
+
+On Linux, installing Podman Desktop with flatpak will detect if the Podman socket exists or not and create it if necessary. That fixes an issue where Podman Desktop was starting but not detecting Podman properly.
+
+**4. Improving consistency of default machine nomenclature**
+
+We have changed the default name of the machine that is initialized from the Resources page of Podman Desktop to maintain the consistency with machines initialized from CLI.
+
+**5. Fixed error handling when pulling images for Desktop Extensions**
+
+If Podman Desktop faces an error while pulling images for Desktop Extensions, the process is terminated and the error is communicated to the user.
+
+![pull image error](img/podman-desktop-release-0.10/pull-image-error.png)
+
+**6. Responsive machine addition or deletion**
+
+When Podman Machine is initialized or deleted, the real-time status is reflected on the menu to the side.
+
+**7. Validate URLs for registries**
+
+The process to validate registry URLs when they are added to Podman Desktop has been improved with the validator library.
+
+**8. Input handling**
+
+Whitespaces in the input for image names are now identified and handled.
+
+## Website and documentation
+
+**1. New Website UI**
+
+As part of our continued efforts to improve the application development journey, we have a new UI for our website which acts as a one-stop-shop for all the information regarding Podman Desktop
+
```

Comment:
the website is not part of the lifecycle of the version
so nothing should appear there.
it's updated as soon as PRs are merged

Reply:
I ended up including these since the PRs for website and docs changes showed up when I filtered the PRs based on the milestone for 0.10.0. Should I go ahead and remove it from the release post?

Reply:
so yes probably

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/abc3da65ea3d6a7da8159e42e4f07fd424eb1655